conanfile.py

```diff
@@ -0,0 +1,94 @@
+from conans import ConanFile, tools
+from os import path
+from six import StringIO
+
+class QbsConanFile(ConanFile):
+    name = "qbs"
+    description = "Conan installer for qbs"
+    topics = ("qmake", "build", "installer")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/qbs/qbs"
+
+    settings = {"os": None, "arch": None, "compiler": None, "build_type": ["Debug", "Release"]}
+    options = {"shared": [True, False]}
+    default_options = {"shared": True}
+
+    def configure(self):
+        self.options["qt"].shared = self.options.shared
+
+    def requirements(self):
+        self.requires("qt/5.15.5")
+
+    def build_requirements(self):
+        if self.settings.os == "Windows":
+            self.tool_requires("jom/1.1.3")
+
+    def source(self):
+        self.run(f"git clone -b v{self.version} --recursive -- https://github.com/qbs/qbs {self.source_folder}")
```

Comment:
It's better to use `tools.Git` I suppose. The best way would be to download archives from qt mirrors and check the sha256 like it's getting done in most of [conan-center-index](https://github.com/conan-io/conan-center-index) recipes using conandata.yml

Reply:
It sounds really better, thanks)